src/actions.js

```diff
@@ -353,7 +359,7 @@ export const fetchNewTxTokenBalance = async (wallet, tx) => {
   const updatedBalanceMap = {};
   const balances = wallet.getTxBalance(tx);
   // we now loop through all tokens present in the new tx to get the new balance
-  for (const [tokenUid, tokenTxBalance] of Object.entries(balances)) {
+  for (const [tokenUid, _tokenTxBalance] of Object.entries(balances)) {
```

Comment:
if `_tokenTxBalance` is never used, wouldn't a `_` be better?

Reply:
I think the effect is the same (making it clear that the variable is unused)

I prefer leaving the variable name there so we know what it is